locustfile.py

```diff
@@ -1,10 +1,11 @@
 import os
 
-from locust import HttpLocust
+from locust import constant, HttpUser
 
 from runner_benchmark.taskset import SurveyRunnerTaskSet
 
 
-class SurveyRunnerScenario(HttpLocust):
-    task_set = SurveyRunnerTaskSet
+class SurveyRunnerScenario(HttpUser):
+    wait_time = constant(0)
```

Comment:
We should review our current use of sleep for a wait time. Happy for this to be done in a separate card.